src/libvirtApi/domain.js

```diff
@@ -502,7 +503,14 @@ export function domainGet({
     let props = {};
     let domainXML;
 
-    return call(connectionName, objPath, 'org.libvirt.Domain', 'GetXMLDesc', [Enum.VIR_DOMAIN_XML_SECURE], { timeout, type: 'u' })
+    const p_getxml = call(connectionName, objPath, 'org.libvirt.Domain', 'GetXMLDesc', [Enum.VIR_DOMAIN_XML_SECURE], { timeout, type: 'u' });
```

Comment:
Just a little nitpick, we don't use this_naming_practive almost anywhere in JS code for variables and functions, only for enums and python tests. Instead thisNamingPractice could fit better

Reply:
Sure! Renamed to `promiseGetXML`.